packages/experimental/src/components/ExampleComponent/ExampleComponent.js

```diff
@@ -53,83 +66,114 @@ export const ExampleComponent = !pkg.isComponentEnabled(componentName)
       return (
         <ButtonSet
           role="main"
-          className={cx([blockClass, `${blockClass}--${size}`, mode])}
+          className={cx(
+            // Apply the block class to the main HTML element, along with
+            // any other classes we need.
+            [blockClass, `${blockClass}--${size}`, modeClass],
+            {
+              // Apply any supplied class names to the main HTML element.
+              [className]: className, // this handles className omitted/falsy
+            }
+          )}
           style={{
             /* stylelint-disable-next-line carbon/theme-token-use */
             [`--${pkg.prefix}-border-color`]: borderColor,
           }}
-          {...props}>
+          {
+            /* Pass through any other property values as HTML attributes. */
+            ...rest
+          }>
           <Button
+            className={`${blockClass}__secondary-button`}
             kind={secondaryKind}
             onClick={handleSecondaryClick}
             size={size}
-            disabled={props.disabled}>
+            disabled={rest.disabled}>
```

Comment:
shouldn't `disabled` just be defined at the top with the other props?

Reply:
Yes, I think so. It looked a bit odd to me too, this. Not sure why it was done. Fixed.